src/main/kotlin/com/mparticle/kits/AppboyKit.kt

```diff
@@ -152,7 +156,11 @@ open class AppboyKit : KitIntegration(), AttributeListener, CommerceListener,
                     event.customAttributeStrings?.let { it ->
                         for ((key, attributeValue) in it) {
                             val hashedKey =
-                                KitUtils.hashForFiltering(event.eventType.value.toString() + event.eventName + key)
+                                if (event.eventName.contains("eCommerce")) {
```

Comment:
this should be comparing the eventType enums because someone could manually put a name with `eCommerce` in it that is not an actual ecommerce event.